exporters/FusionExporter/Source/AddIn/CustomHandlers.h

```diff
@@ -50,6 +50,7 @@ namespace SynthesisAddIn
 		ShowPaletteCommandCreatedHandler(EUI* eui, std::string id_) : eui(eui), id(id_) {  }
 		~ShowPaletteCommandCreatedHandler();
 		void notify(const Ptr<CommandCreatedEventArgs>& eventArgs) override;
+		void setId(std::string id_) { id = id_; }
```

Comment:
Does the handler's command ID ever need to be modified after the handler is constructed?